codemp/client/cl_console.cpp

```diff
@@ -478,19 +479,22 @@ void CL_ConsolePrint( const char *txt) {
 		}
 
 		// count word length
-		for (l=0 ; l< con.linewidth ; l++) {
-			if ( txt[l] <= ' ') {
-				break;
+		if ((l < 0 ? l = 0, true : false) || (l + 1 == con.linewidth)) {
```

Comment:
If length is ever less than 0, set it to 0 and return true, else return false. So if it's less than 0, it'll count word length but start with txt[0].
(l + 1 == con.linewidth) this is to continue counting word length since the while loop stops when the length is == con.linewidth